src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -58,6 +58,7 @@
     public static final String MESSAGE_EDIT_PERSON_SUCCESS = "Edited Person: %1$s";
     public static final String MESSAGE_NOT_EDITED = "At least one field to edit must be provided.";
     public static final String MESSAGE_DUPLICATE_PERSON = "This person already exists in the address book.";
+    public static final String MESSAGE_SAME_STATUS = "Edited covid status is same as the current one";
```

Comment:
I think your function is checking if any of the fields of the edited person is the same. Therefore, the message shouldn't be "Edited covid status" but maybe more of "The attribute you want to edit is the same?". Maybe you can find a way to phrase it better 👍

Reply:
It is a great suggestion! I have changed it to make it more understandable.